amida-tech/indaba-client
# Feature/inba 662 survey edit styles

If this PR fixes a bug, you _must_ add test cases representative of the bug.

Please refer to [Tettra](https://app.tettra.co/teams/amida/pages/amida-pull-request-and-code-review-guide) for PR review guidelines.

#### What does this PR do?
Styles the survey modal, adds green hover to the icons in the status card
#### Related JIRA tickets:
https://jira.amida-tech.com/browse/INBA-662
#### How should this be manually tested?
Go to a project
Hover over survey name. Check that its green. Click on the survey name, should trigger modal, change the name and save. Ensure the name changed. 
change the name
#### Background/Context

#### Screenshots (if appropriate):
